src/component/test-component.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-const DependantConfigBasedComponent = require('./dependant-config-based-component');
-const emitEvents = require('./emit/events');
-const events = require('./test/events');
 const print = require('print');
+const unitEvents = require('./test/events');
+const emitEvents = require('./emit/events');
+const UnitRunner = require('./test/unit-runner');
 const ContainerTransformer = require('./helper/container-transformer');
-const Mocha = require('mocha');
+const DependantConfigBasedComponent = require('./dependant-config-based-component');
```

Comment:
noun: dependant
a person who relies on another, especially a family member, for financial support.

adjective: dependent
contingent on or determined by.

Let's rename this from "dependant" to "dependent" ;)